### PR TITLE
feat: add Fancy Emoji Python plugin (Wox port)

### PR DESCRIPTION
**Pull request by Ma-ve**
Context: https://github.com/Flow-Launcher/Flow.Launcher/discussions/512#discussioncomment-961575

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -226,5 +226,16 @@
...
         "E-tag": "W/\"79887f237c773da672443ae2e4a99f1334d6f53b607eeb9dc1dfb37a3ef329fb\""
+    },
+    {
+        "ID": "FF2C31D6A47348FFB9ED4EB26F6794E2",
+        "Name": "Fancy Emoji",
+        "Description": "Search for emoji and add to your clipboard automatically",
+        "Author": "Mave",
+        "Version": "1.0.1",
+        "Language": "python",
+        "Website": "https://github.com/Ma-ve/Flow.Launcher.PluginsManifest",
```
Comment (jjw24) [inline plugins.json:237]: this website doesnt look right. you can set it the same url as the repo's url

File: plugins.json [hunk 2]
```diff
@@ -226,5 +226,16 @@
...
+        "UrlDownload": "https://github.com/Ma-ve/Flow.Launcher.Plugin.FancyEmoji/releases/download/1.0.1/Flow.Launcher.Plugin.FancyEmoji.zip",
+        "UrlSourceCode": "https://github.com/Ma-ve/Flow.Launcher.PluginsManifest/tree/master"
```
Comment (jjw24) [inline plugins.json:239]: this url also doesnt look right

**Comment by jjw24** [comment]
could you enable issues for the project please

File: plugins.json (hunk 1 above)
Comment (Ma-ve) [reply plugins.json:237]: What do you mean, doesn't look right? I can or I can't set it to the same URL?

**Comment by Ma-ve** [comment]
> @jjw24 
> 
> could you enable issues for the project please

Enabled!

File: plugins.json (hunk 2 above)
Comment (Ma-ve) [reply plugins.json:239]: How come? It's the exact same as the two plugins above: RollDice and isPrime

File: plugins.json (hunk 1 above)
Comment (jjw24) [reply plugins.json:237]: i mean the two urls do not resolve to anything if you open https://github.com/Ma-ve/Flow.Launcher.PluginsManifest. I think you meant to put in https://github.com/Ma-ve/Flow.Launcher.Plugin.FancyEmoji right?

File: plugins.json (hunk 2 above)
Comment (jjw24) [reply plugins.json:239]: this is the source code of your plugin

**Comment by jjw24** [comment]
> > @jjw24
> > could you enable issues for the project please
> 
> Enabled!

i meant the issues for https://github.com/Ma-ve/Flow.Launcher.Plugin.FancyEmoji

File: plugins.json (hunk 1 above)
Comment (Ma-ve) [reply plugins.json:237]: Oooh, apologies -- how did I miss that!? Thanks!

**Comment by Ma-ve** [comment]
> 
> 
> > > @jjw24
> > > could you enable issues for the project please
> > 
> > 
> > Enabled!
> 
> i meant the issues for [Ma-ve/Flow.Launcher.Plugin.FancyEmoji](https://github.com/Ma-ve/Flow.Launcher.Plugin.FancyEmoji)

😂 

done

File: plugins.json (hunk 2 above)
Comment (Ma-ve) [reply plugins.json]: Changed!

File: plugins.json (hunk 1 above)
Comment (Ma-ve) [reply plugins.json]: Edited!

Comment (jjw24) [reply plugins.json]: > Oooh, apologies -- how did I miss that!? Thanks!

All good

**Review by jjw24 — APPROVED** [review]

**Pull request merged**